src/EnergyPlus/OutputReportTabular.hh

```diff
@@ -683,13 +685,24 @@ namespace OutputReportTabular {
 
     void LoadSummaryUnitConversion(EnergyPlusData &state, CompLoadTablesType &compLoadTotal);
 
+    void LoadSummaryUnitConversion(EnergyPlusData &state, CompLoadTablesType &compLoadTotal, iUnitsStyle unitsStyle_para);
+
     void CreateListOfZonesForAirLoop(EnergyPlusData &state, CompLoadTablesType &compLoad, Array1D_int const &zoneToAirLoop, int const &curAirLoop);
 
     void OutputCompLoadSummary(EnergyPlusData &state,
                                iOutputType const &kind,
                                CompLoadTablesType const &compLoadCool,
                                CompLoadTablesType const &compLoadHeat,
-                               int const &zoneOrAirLoopIndex);
+                               int const &zoneOrAirLoopIndex,
+        iUnitsStyle unitsStyle_para, bool produceTabular_para, bool produceSQLite_para);
+
+    //void OutputCompLoadSummary(EnergyPlusData &state,
+    //                           iOutputType const &kind,
+    //                           CompLoadTablesType const &compLoadCool,
+    //                           CompLoadTablesType const &compLoadHeat,
+    //                           int const &zoneOrAirLoopIndex,
+    //                           iUnitsStyle unitsStyle_para,
+    //                           bool produceSQLite_para);
```

Comment:
Weird indentation and also stray commented block left in here.

Reply:
This is corrected now. Also removed the redundant commented-out one.